namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-great-ideas-uat/resources/main.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=1.0"
```

Comment:
you will want ref=2.0 here, we have a new release: https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
Thanks, now amended, and I'll paste the PR in the channel next time 👍